v5/heroku.go

```diff
@@ -415,7 +421,8 @@ func (s *Service) AddOnListByApp(ctx context.Context, appIdentity string, lr *Li
 }
 
 type AddOnUpdateOpts struct {
-	Plan string `json:"plan" url:"plan,key"` // unique identifier of this plan
+	Name *string `json:"name,omitempty" url:"name,omitempty,key"` // globally unique name of the add-on
```

Comment:
At least, I finally got the field I originally wanted.

Reply:
🍻